ui/components/multichain/account-list-item/account-list-item.js

```diff
@@ -153,7 +153,7 @@ export const AccountListItem = ({
                 as="button"
                 onClick={(e) => {
                   e.stopPropagation();
```

Comment:
Should propagation bubble upwards if there's no `onClick`?

Reply:
I'm not sure. I would prefer to not alter this component further, we are simply reusing it for display purposes with clicks disabled.